### ISSUE TITLE
Trying to redirect terraform output when using `atmos` doesn't seem possible.

### ISSUE THREAD

**Issue by Gowiem** [cloudposse/atmos#203]
## Describe the Feature

It is not possible to write the output of a particular terraform command (e.g. `terraform output`) to a file. Doing something like:

```shell
atmos terraform output aws-users -s identity --json > test.json
```

Will output the entire atmos command output instead of just the `terraform output` output. 

Trying `atmos terraform output aws-users -s identity "--json > test.json"` also does not work with the following error: 

```
Executing command:
/usr/bin/terraform output --json > test.json
╷
│ Error: Failed to parse command-line flags
│
│ flag provided but not defined: -json > test.json
╵

For more help on using this command, run:
  terraform output -help
exit status 1
```

I would like to talk through this issue on what can be done. My first suggestion is that we add a new global CLI option `--output` that will output the last (usually the most important command) to the a provided filename. 

## Expected Behavior

The below command (with new `--output` option) would run the atmos commands as normal, but write the output of the `terraform output` command to `test.json`. 

```shell
atmos terraform output aws-users -s identity --json --output test.json
```

## Use Case

When you need to collect output information from a particular atmos subcommand. 

## Alternatives Considered

I don't have any, but would be interested to discuss if there are any that might make sense. 

## Additional Context

N/A

**Comment by nitrocode** [comment]
As a workaround, have you tried appending `2>&1` ?

```sh
atmos terraform output aws-users -s identity --json > test.json 2>&1
```

**Comment by Gowiem** [comment]
@nitrocode just tried. It resulted in the full atmos output + `terraform output`. 

I can only imagine redirecting stderr to `/dev/null` via `2>/dev/null` would possibly work if `atmos` was writing all of its logs to `stderr`... but even then the `terraform init` output would still be sent to `stdout` unless that was redirected as well.

**Comment by aknysh** [comment]
@Gowiem I don't know if it's possible to do in `Go` (I guess not w/o some additional command-line flags) since atmos outputs all the messages to stdout, then calls `terraform` which outputs to stdout as well - not sure how to separate those
And we don't want to output messages to stderr (this is not right).

As a workaround, try to use `atmos terraform shell`

https://github.com/cloudposse/atmos/pull/103

```
After executing atmos terraform shell <component> -s <stack> command, the CLI generates all the required variables for the component in the stack, create a Terraform workspace, and drops the user into a separate shell (process) with all the required paths and ENV vars set, allowing execution of all native Terraform commands using the native syntax
```

Execute the command, then you are free to execute any native TF command w/o atmos involved.

**Comment by Gowiem** [comment]
@aknysh yeah, I think it would require an additional command-line flag that would then result in the last terraform command being executed with `stdout` redirection to the command-line flag's value. e.g. `--output test.json` would result in `terraform output --json > test.json`. 

Definitely agreed on not outputting messages to `stderr` 👍 

`atmos terraform shell` would work in the case of doing this interactively, but I don't think it would work in my use-case here which is executing `atmos terraform output` from a script. Because `shell` starts a subshell, I imagine that would not work out... but I'll test to confirm.

**Comment by aknysh** [comment]
in any case (whether `terraform shell` works for you or not), we can add a flag to all atmos commands to silence it (so only the underlying tools will output to stdout)

**Comment by Gowiem** [comment]
@aknysh I don't see how I would execute `atmos terraform shell` in a scripted sense, so can't test it. If you have any linux-fu ideas that would pass a command down to a subshell then let me know. 

As for a new flag to silence so only the underlying tools that are executed, that would likely only get us part of the way there because `terraform init` and `terraform workspace select` outputs would still end up in the resulting `stdout` redirection, which means we can't just collect the output of the last command (`terraform output` in my case).

**Comment by aknysh** [comment]
ok, i'll think about what could be done here

**Comment by osterman** [comment]
I think a flag option that @Gowiem suggests makes the most sense. Maybe it should be qualified to `--output-file` to be clear it's not the type of output, but a destination.

**Comment by Gowiem** [comment]
Ah @Nuru brought up a similar issue in #140 that I'm just now seeing.

**Comment by dudymas** [comment]
Hey there, working on `Refarch-Scaffold` , and we (Igor and I) are noticing that it's tricky to get atmos to write vpn configs.
Lines like [this one](https://github.com/cloudposse/atmos/blob/89a2ad0adc4b1d121e5fc8c368976ce46928d138/internal/exec/terraform.go#L293) should likely go to a log handling library, so that end users can specify things like `LOG_LEVEL=INFO` and we could filter out the other details of the command as being on `LOG_LEVEL=DEBUG`

Note the quick way is just to add a flag to let the user specify a file, or to support the `-c` flag when running `atmos terraform shell`

**Comment by aknysh** [comment]
@Gowiem  this was fixed in https://github.com/cloudposse/atmos/releases/tag/v1.33.0

**Comment by Gowiem** [comment]
Thanks for the heads up @aknysh!

Cc @kevcube

**Comment by kevcube** [comment]
Woo thanks @aknysh!
